File: app/src/main/java/com/example/diploma/MainMenu.java
package com.example.diploma;

import android.os.Bundle;
import android.view.MenuItem;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import com.example.diploma.MainRecycler.Divisions;
import com.example.diploma.MainRecycler.MainFragment;
import com.example.diploma.news.NewsFragment;
import com.example.diploma.profile.ProfileFragment;
import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.google.android.material.navigation.NavigationBarView;

import java.util.ArrayList;

public class MainMenu extends AppCompatActivity {

    ArrayList<Divisions> states = new ArrayList<Divisions>();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_menu);
        BottomNavigationView bottomNavigationView = (BottomNavigationView)
                findViewById(R.id.bottom_boot);

        startFragment(new MainFragment());


        bottomNavigationView.setOnItemSelectedListener(new NavigationBarView.OnItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                switch (menuItem.getItemId()) {
                    case R.id.action_map:
                        startFragment(new NewsFragment());
                        return true;
                    case R.id.action_dial:
                        startFragment(new MainFragment());
                        return true;
                    case R.id.action_mail:
                        startFragment(new ProfileFragment());
                        return true;
                }
                return false;
            }
        });
    }

    private void startFragment(Fragment fragment) {
        FragmentManager manager = getSupportFragmentManager();
        FragmentTransaction ft = manager.beginTransaction();
        ft.replace(R.id.fragment_container, fragment);
        ft.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE);
        ft.commit();

    }
}

File: app/src/main/java/com/example/diploma/view/main/MainMenu.kt
package com.example.diploma.view.main

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentTransaction
import com.example.diploma.R
import com.example.diploma.view.news.NewsFragment
import com.example.diploma.view.profile.ProfileFragment
import com.google.android.material.bottomnavigation.BottomNavigationView
import com.google.android.material.navigation.NavigationBarView

class MainMenu : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_menu)
        val bottomNavigationView = findViewById<View>(R.id.bottom_boot) as BottomNavigationView

        startFragment(MainFragment())


        bottomNavigationView.setOnItemSelectedListener(NavigationBarView.OnItemSelectedListener { menuItem ->
            when (menuItem.itemId) {
                R.id.action_map -> {
                    startFragment(NewsFragment())
                    return@OnItemSelectedListener true
                }

                R.id.action_dial -> {
                    startFragment(MainFragment())
                    return@OnItemSelectedListener true
                }

                R.id.action_mail -> {
                    startFragment(ProfileFragment())
                    return@OnItemSelectedListener true
                }
            }
            false
        })
    }

    private fun startFragment(fragment: Fragment) {
        val manager = supportFragmentManager
        val ft = manager.beginTransaction()
        ft.replace(R.id.fragment_container, fragment)
        ft.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
        ft.commit()
    }
}